image/options.go

```diff
@@ -47,8 +45,12 @@ type Options struct {
 	// TODO: use OptionsSnap directly here?
 	Snaps        []string
 	SnapChannels map[string]string
-	Revisions    map[string]snap.Revision
 
+	// SeedManifest is a pre-provided seed manifest, to allow for
+	// creating reproducible seeds. If provided, the snap revisions and
+	// validation-sets specified in the seed manifest will be used to
+	// (re-)create the image seed.
+	SeedManifest *SeedManifest
```

Comment:
This change requires updating the usage in ubuntu-image, a PR i'll open when it lands